_posts/2022-10-28-multiple-data-source.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title:  "Launch Highlight: Multiple Data Sources"
+authors:
+  - tyarong
+  - zengyan
+  - kroosh
+date:   2022-10-28 00:00:00 -0700
+categories:
+  - community
+---
+
+OpenSearch Dashboards' current architecture works only with a single OpenSearch cluster. To view data in different OpenSearch clusters and to navigate between Dashboards endpoints, users must have a complete OpenSearch with Dashboards stack setup. To solve this challenge, we are excited to announce for release 2.4 the multiple data sources feature. This experimental feature supports multiple data source connections in OpenSearch Dashboards.
```

Comment:
In rereading the first sentence, it flows better if we make a slight change as follows:

"...with a single OpenSearch cluster**, and** to view data in..."